include/sprite.h

```diff
@@ -216,16 +216,13 @@ typedef struct {
     /* 0x28 */ Sprite_UNK28 unk28[1];
 } Sprite /* size = 0x30 */;
 
-// Transformer
-struct UNK_808D124_UNK180 {
-    u16 unk0;
-    // width
-    s16 unk2;
-    // height
-    s16 unk4;
-    // pos
-    s16 unk6[2];
-}; /* size 0xA */
+typedef struct {
+    /* 0x00 */ u16 unk0;
+    /* 0x02 */ s16 width;
+    /* 0x04 */ s16 height;
+    /* 0x06 */ s16 x;
```

Comment:
Interesting. I think at some point I could only match with an array, but I guess this was resolved

Reply:
I tried Vector_16, but that sadly didn't match.